src/Engine/RadiumEngine.cpp

```diff
@@ -43,7 +43,7 @@ RadiumEngine::~RadiumEngine() = default;
 
 void RadiumEngine::initialize() {
     LOG( logINFO ) << "*** Radium Engine ***";
-    m_resourcesRootDir    = {Core::Resources::getRadiumResourcesDir()};
+    m_resourcesRootDir    = {Core::Resources::getRadiumResourcesPath()};
```

Comment:
Not sure that `={}`is needed. Why not just an `=` or a `std::move()` ...
This is legal, and used several time in other files/functions, but I do not understand the need to cast explicitly and then force the assignment as the return value of `Core::Resources::getRadiumResourcesPath()` is of the same type as the attribute `m_resourcesRootDir`

Reply:
Well, it seems it's not related to this PR, but yes, upcoming commit fix that
Feel free to fix all of them in the codebase :rofl:

Reply:
🥵 Well, its better that everybody meeting one of these always fix it. Like this, the code will be asymptotically clear 🤣